MetaMask/metamask-extension
# Adopt new version of `@metamask/signature-controller` to add signature operation logs

## **Description**

This PR aims to update `@metamask/signature-controller@^6.1.2` in order to adopt add logs on signature operations.

## **Manual testing steps**

There is no functional change in signatures. But if you get signature request, sign or rejects, that should end up in `MetaMask State Logs.json` which could be downloaded under `Settings>Download State Logs`

## **Screenshots/Recordings**

I am adding an example of state logs as a proof. (Please note that I've removed locale messages to make file a bit smaller)
[MetaMask state logs.txt](https://github.com/MetaMask/metamask-extension/files/12816449/MetaMask.state.logs.txt)

## **Related issues**

_Fixes https://github.com/MetaMask/MetaMask-planning/issues/1321

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
